tools/util/include/cutlass/util/device_memory.h

```diff
@@ -58,11 +58,14 @@ T* allocate(size_t count = 1) {
 
   bytes = count * sizeof(T);
 
+#ifndef CUTLASS_ENABLE_SYCL
```

Comment:
On other situations we always used #ifdef. for synchronicity can you follow the same pattern